common/djangoapps/cme_registration/models.py

```diff
@@ -103,14 +103,13 @@ class Meta:
     sub_specialty = models.CharField(blank=True, null=True, max_length=255)
 
     AFFILIATION_CHOICES = (('Stanford Children\'s Health', 'Stanford Children\'s Health'),
-                           ('Packard Children\'s Health Alliance', 'Packard Children\'s Health Alliance'),
+                           ('Packard Children\'s Health Alliance (PCHA)', 'Packard Children\'s Health Alliance (PCHA)'),
                            ('Stanford Health Care', 'Stanford Health Care'),
                            ('Stanford University', 'Stanford University'),
-                           ('University Healthcare Alliance', 'University Healthcare Alliance'),
-                           ('Other', 'Other'),
+                           ('University Healthcare Alliance (UHA)', 'University Healthcare Alliance (UHA)'),
+                           ('Not affiliated with Stanford Medicine', 'Not affiliated with Stanford Medicine'),
                            )
     affiliation = models.CharField(blank=True, null=True, max_length=46, choices=AFFILIATION_CHOICES)
-    other_affiliation = models.CharField(blank=True, null=True, max_length=46)
```

Comment:
We shouldn't be removing this field as it contains data for existing registrants which would be deleted and who knows, they may ask for it back in a few weeks. Additionally, there's a batch script that runs that slurps up the cme data that may be expecting the column.

I think the point of the Trello card is just to change the UI behaviour.